shiny/BrownDog/server.R

```diff
@@ -8,10 +8,14 @@ library(PEcAn.visualization)
 # Define server logic
 server <- shinyServer(function(input, output, session) {
   output$typeSelector <- renderUI({
-    bety <- betyConnect("../../web/config.php")
-    con <- bety$con
-    on.exit(db.close(con))
-    datatype <- db.query("SELECT name FROM sitegroups;", con)
+    # get the files in library PEcAn.data.atmosphere under registration
+    libPaths<-.libPaths()
+    index <- dir.exists(paste0(libPaths, '/PEcAn.data.atmosphere'))
+    data.atmosphere.registration <-paste0(libPaths[index], "/PEcAn.data.atmosphere/registration")
```

Comment:
done. 

but it is "Ameriflux" & "Fluxnet2015". 

I should map them to AmeriFlux or change  agreement, to 

"Ameriflux" = checkboxInput("agreement", HTML("I agree to <a href='http://ameriflux.lbl.gov/data/data-policy/'>AmeriFlux license</a>."), value = FALSE, width = NULL)

?